src/pages/_app.tsx

```diff
@@ -6,37 +6,14 @@ import 'react-toastify/dist/ReactToastify.css';
 
 import { SEO } from '@/components/common';
 import { PageLayout } from '@/components/layout';
-import { CookieService } from '@/lib/services';
-import type { PrivateProfile } from '@/lib/types/apiResponses';
-import { CookieType } from '@/lib/types/enums';
-import { NextPageContext } from 'next';
 import { ThemeProvider } from 'next-themes';
 import type { AppProps } from 'next/app';
 import { DM_Sans as DMSans } from 'next/font/google';
-import { useEffect, useState } from 'react';
 import { ToastContainer } from 'react-toastify';
 
-interface InitialPropInterface {
-  user?: PrivateProfile;
-}
 const dmSans = DMSans({ subsets: ['latin'], weight: ['400', '500', '700'] });
 
-export default function MyApp({ Component, pageProps }: AppProps<InitialPropInterface>) {
-  const [user, setUser] = useState(pageProps?.user);
-
-  // For 404 page: Try getting user from cookie on client side (because 404
-  // pages in Next.js must be a static page)
-  useEffect(() => {
-    if (pageProps?.user) {
-      setUser(pageProps?.user);
-      return;
-    }
-    const userCookie = CookieService.getClientCookie(CookieType.USER);
-    if (userCookie) {
-      setUser(JSON.parse(userCookie));
-    }
-  }, [pageProps?.user]);
```

Comment:
regression: removing this line undoes a change (which came from the storefront PR) that adds the signed-in navbar to the 404 page

main branch (for https://v2.members.acmucsd.com/store/collection/aca1d17d-7002-46fb-808c-f080703edd4c):

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/5e1a5dd1-2673-4fcc-a346-bfc8426cb4c5)


this branch:

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/84f5a87b-67ef-4bc1-b390-5d75e1c3b01d)